v1/proto/service/gribi.proto

```diff
@@ -421,3 +427,64 @@ message GetResponse {
   // A set of entries that are installed on the device.
   repeated AFTEntry entry = 1;
 }
+
+// FlushRequest is the request sent from the client to the server when a Flush
+// RPC is called.
+message FlushRequest {
+  // election specifies the election ID from a client. It is valid only
+  // when the server has clients that are operating in the SINGLE_PRIMARY mode.
+  oneof election {
+    // id indicates that the Flush ID supplied should be compared to the
+    // existing election ID on the server. The logic
+    // for whether the Flush RPC is accepted is the same as that which is
+    // implemented for Modify. In summary:
+    //
+    //   - if the election ID is equal to the previous highest known ID, the
+    //     Flush is accepted.
+    //   - if the election ID is greater than the previous highest known ID,
+    //     the Flush is accepted.
+    //   - if the election ID is less than the previous highest known ID, the
+    //     Flush is rejected and an error with reason NOT_PRIMARY is returned.
+    //
+    // If the server is currently operating in ALL_PRIMARY mode the server
+    // should return an error specifying ELECTION_ID_IN_ALL_PRIMARY reason.
+    Uint128 id = 1;
+    // override indicates that the client explicitly requests that the election
+    // ID should not be compared, regardless of whether there are SINGLE_PRIMARY
+    // clients on the server.
+    bool override = 2;
+  }
+  // network_instance specifies the Network Instance that should be flushed.
+  oneof network_instance {
+    // The network instance from which the entries should be flushed.
+    // If name is specified the empty string "" is considered invalid.
+    string name = 3;
+    // all indicates that entries for all network instances for the
+    // specified network instance should be flushed.
```

Comment:
There's an extraneous `for the specified network instance`.

```
all indicates that entries for all network instances should be flushed.
```

Reply:
Thanks, this was both a copy-pasta and an error in `Get`. Fixed.